test/e2e/find-bridge-vlan-centos-8.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -e
```

Comment:
The script is named "find-bridge-vlan-centos-8". Not sure I understand the "find" part there. What about `check-has-vlans-el8.sh`? It look very much alike `func hasVlans(node string, connection string, minVlan int, maxVlan int) AsyncAssertion`.